packages/pyright-internal/src/analyzer/typeDocStringUtils.ts

```diff
@@ -64,6 +64,7 @@ function isInheritedFromBuiltin(type: FunctionType | OverloadedFunctionType, cla
         type.details.moduleName === 'builtins' &&
```

Comment:
Should this be calling `isBuiltIn` rather than checking the `moduleName`? Not sure if this is intentional or just old.